app/src/main/java/org/p2p/wallet/striga/onramp/iban/presenter/StrigaIbanImportantNotesDialogPresenter.kt

```diff
@@ -0,0 +1,33 @@
+package org.p2p.wallet.striga.onramp.iban.presenter
+
+import org.p2p.wallet.R
+import org.p2p.wallet.common.mvp.BasePresenter
+import org.p2p.wallet.striga.onramp.iban.StrigaIbanImportantNotesContract
+import org.p2p.wallet.striga.onramp.interactor.StrigaOnRampInteractor
+
+class StrigaIbanImportantNotesDialogPresenter(
```

Comment:
also i can suggest injecting just a mapper inside bottom sheet, i think presenter for this is a little bit of overhead

Reply:
there is also the OnRampInteractor being used